main.js

```diff
@@ -0,0 +1,39 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+let render = function () { renderer.renderPosts(tweeter.getPosts()) }
+
+const post = function () {
+    const inpval = $("#input").val()
```

Comment:
Name your variable `inputVal` instead. Don't need to save on the extra two letters.